react/SeekApp/SeekApp.js

```diff
@@ -2,23 +2,46 @@ import styles from './SeekApp.less';
 
 import React, { PropTypes } from 'react';
 import classnames from 'classnames';
+import Helmet from 'react-helmet';
+import ScreenReaderOnly from 'Accessibility/ScreenReaderOnly';
 
-export default function SeekApp({ fullScreen, children }) {
+const defaultPageTitleAU = 'SEEK - Australia\'s no. 1 jobs, employment, career and recruitment site';
+const defaultPageTitleNZ = 'Jobs on SEEK - New Zealand\'s no. 1 Employment, Career and Recruitment site';
+
+const getLocalisedPageTitle = locale => locale === 'AU' ? defaultPageTitleAU : defaultPageTitleNZ;
+
+export default function SeekApp({ fullScreen, children, meta, link, title, locale }) {
```

Comment:
This could just be an object for `Helmet` but I thought making these explicit might be better?